web/templates/message_avatar.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="u-{{msg/sender_id}} message-avatar sender_info_hover view_user_card_tooltip" aria-hidden="true" data-tooltip-template-id="view-user-card-tooltip-template">
-    <div class="inline_profile_picture {{#sender_is_guest}} guest-avatar{{/sender_is_guest}}">
+    <div class="inline_profile_picture {{#sender_is_guest}} guest-avatar {{/sender_is_guest}}{{#sender_is_bot}} bot-avatar {{/sender_is_bot}}">
```

Comment:
Here we would have both `guest-avatar` and `bot-avatar` class for bots whose role is guest. While in other places where avatar is shown, we would have `guest-avatar` for such case. This should also be made consistent with other places.

Reply:
@sahil8399
Can you check it again?.